larq/quantized_variable_test.py

```diff
@@ -85,9 +85,10 @@ def test_method_delegations(distribute_scope):
             # These attributes are not supported for DistributedVariables
             assert x.constraint is None
             assert x.initializer == x.latent_variable.initializer
-        assert evaluate(x.assign(4)) == 8
-        assert evaluate(x.assign_add(1)) == 10
-        assert evaluate(x.assign_sub(1.5)) == 7
+            # DistributedVariables.assign doesn't correctly return quantized variables
+            assert evaluate(x.assign(4)) == 8
+            assert evaluate(x.assign_add(1)) == 10
+            assert evaluate(x.assign_sub(1.5)) == 7
```

Comment:
Wouldn't it be nicer to split the assign and read instead of skipping?
```suggestion
        
        def apply_and_read(x, fn, args):
            evaluate(fn(*args))
            return evaluate(x)

        assert apply_and_read(x, x.assign, [4]) == 8
        assert apply_and_read(x, x.assign_add, [1]) == 10
        assert apply_and_read(x, x.assign_sub, [1.5]) == 7
```